torch/distributed/elastic/timer/file_based_local_timer.py

```diff
@@ -78,10 +78,10 @@ class FileTimerClient(TimerClient):
         signal: singal, the signal to use to kill the process. Using a
                         negative or zero signal will not kill the process.
     """
-    def __init__(self, file_path: str, signal=signal.SIGKILL) -> None:
+    def __init__(self, file_path: str, signal: Optional[_signal.Signals] = None) -> None:
         super().__init__()
         self._file_path = file_path
-        self.signal = signal
+        self.signal = _signal.SIGKILL if signal is None else signal
```

Comment:
Should this be used instead for windows support? https://github.com/pytorch/pytorch/blob/8e1ae1c19d7096e72d5e095a7c5de0acf05c5fbf/torch/distributed/elastic/multiprocessing/api.py#L65

Reply:
Oh awesome! I didn't know this exists. I think we should definitely use this :) Updated!